### PR TITLE
Fix/consumer-integration-tests

### PR DESCRIPTION
**Pull request by vagnersaraiva**
- test(integration): fixing consumer integration test, executing publish process before testing

### CODE REVIEW

File: src/__tests__/int/consumer.int.spec.ts [hunk 1]
```diff
@@ -1,10 +1,20 @@
-import { Subscribe } from "../..";
+import { Publish, Subscribe } from "../..";
 jest.unmock("kafkajs");
 
 describe("Consumer Integration Tests", () => {
-  const propertiesResponse = ["key", "message", "offset", "partition", "topic"];
+  beforeAll(async () => {
```
Comment (robertLichtnow) [inline src/__tests__/int/consumer.int.spec.ts]: You should add an afterAll method that clears the queue, due to when the test fails, the queue stays empty

File: src/__tests__/int/consumer.int.spec.ts [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+    const publish = Publish();
+    await publish.send("topic1", "hello world");
```
Comment (robertLichtnow) [inline src/__tests__/int/consumer.int.spec.ts]: Change the topic name to a differente name from the publisher one, like "test.consumer.mock.topic"

**Comment by robertLichtnow** [comment]
Change the publisher test topic name to "test.publisher.mock.topic"

**Comment by robertLichtnow** [comment]
The publisher test still leaves messages on the queue https://github.com/natura-cosmeticos/iris-nodejs-messenger-kafka/blob/master/src/__tests__/int/producer.int.spec.ts

**Review by robertLichtnow — CHANGES_REQUESTED** [review]

**Review by robertLichtnow — APPROVED** [review]
